source/jormungandr/jormungandr/interfaces/v1/serializer/pt.py

```diff
@@ -208,6 +208,10 @@ def get_stop_point(self, obj):
 class ImpactedSectionSerializer(PbNestedSerializer):
     f = PtObjectSerializer(label='from', attr='from')
     to = PtObjectSerializer()
+    routes = jsonschema.MethodField(schema_type=lambda: RouteSerializer(Many=True))
+
+    def get_routes(self, obj):
+        return RouteSerializer(obj.routes, display_none=False, many=True).data
```

Comment:
Why not just

```python
routes = RouteSerializer(display_none=False, many=True)
```

Reply:
Because `RouteSerialiser` isn't yet defined (it's on line 370, so after this)